packages/angular_devkit/build_angular/test/differential-loading/is-differential-loading-needed_spec_large.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
This seems to be a `spec` file which can be combined with the others.

Reply:
Which others do you mean?

Reply:
This one https://github.com/angular/angular-cli/pull/13993/files#diff-7379beedd1dbd6738b8da9124058d8b2

Reply:
Ah, I see. It's about merging the `spec` with the `large_spec` file, right? 

Currently, `spec` does everything we can do in memory. If I need to access the hard disk, I'm using `large_spec`. Isn't this the intended way?

Reply:
Typically test large are used to test the full architect builder and not a unit.

Reply:
Ok, I see. In this case, merging makes sense.